homestyle.css

```diff
@@ -90,33 +78,30 @@ a:active{
     top: 70%;
     left: 70%;
     transform: translate(-50%, -50%);
-	color: white;
-	font-weight:lighter;
-	text-family: 'Papyrus', sans-serif;
+	font-weight: 100;
+	font-family: 'Noto Sans KR', sans-serif;
 }
 
 /* About Me */
 
-.aboutmeparent1 {
+.aboutmecontainer {
 	width: auto;
 	display: flex;
 	flex-direction: row;
```

Comment:
This is the automatic flex direction unless you have previously specified it elsewhere as being different?